spec/requests/as_account/playlist_spec.rb

```diff
@@ -206,7 +206,11 @@
   end
 
   context 'given one of my own playlists that I want to get reports for' do
-    let(:id) { $account.channel.playlists.first.id }
+    let(:id) { test_account.channel.playlists.first.id }
+
+    before do
+      allow(Date).to receive(:today).and_return(Date.new(2020, 2, 5))
```

Comment:
why do we need this?

Reply:
Because this test is exercising a function that (ultimately) calls `Date.today`. We need to fix that so that it's deterministic for VCR (otherwise it'll see a request for a different day and fail).